modules/customize-certificates-understanding-default-router.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+//
+// authentication/certificates/replacing-default-ingress-certificate.adoc
+
+[id="understanding-default-ingress-{context}"]
+= Understanding the default ingress certificate
+
+By default {product-title} uses the Ingress Controller Operator to 
+create an internal CA and issue a wildcard certificate that is valid for 
+applications under the `.apps` sub-domain. Both the web console and CLI
+use this certificate as well. 
+
+The internal infrastructure certificates are self-signed. 
+While this process might be perceived as bad practice by some security or 
+PKI teams, any risk here is minimal. The only clients that trust these 
+certificates are other components within the cluster. If you replace 
+the default wildcard certificate, then all external users and systems
+will use the newly defined certificate.
```

Comment:
Saying "external users and systems will *use*" is incorrect. It should say something more along the lines of:

"Replacing the default wildcard certificate with one that is issued by a public or organizational CA will allow external clients to connect securely to applications running under the apps sub-domain"

Reply:
Updated the last sentence to use this sentence.